pkg/controller/ps/user.go

```diff
@@ -224,7 +233,19 @@ func (r *PerconaServerMySQLReconciler) reconcileUsers(ctx context.Context, cr *a
 	}
 	log.V(1).Info("Got primary host", "primary", primaryHost)
 
-	um, err = users.NewManager(apiv1alpha1.UserOperator, operatorPass, primaryHost, mysql.DefaultAdminPort)
+	// TODO: handle this differently
+	// Because how currently we implement manager.DiscardOldPasswords,
+	// (as well ass manger.UpdateUserPasswords), we need updated operator user pass
+	// to perform DiscardOldPasswords properly.
```

Comment:
This is directly related to this https://github.com/percona/percona-server-mysql-operator/pull/386#discussion_r1253071485